### PR TITLE
Mention 'main' instead of 'master'

### PR DESCRIPTION
**Pull request by dougbu**
- part of #29475
- primarily involves updating links to this repo

nit: Remove broken links to old Travis environments (that happen to mention 'master')

### CODE REVIEW

**Review by BrennanConroy — APPROVED** [review]

File: src/SignalR/clients/java/signalr/core/build.gradle [hunk 1]
```diff
@@ -78,4 +78,4 @@ class Version {
     }
 }
 
-compileJava.dependsOn generateVersionClass
\ No newline at end of file
+compileJava.dependsOn generateVersionClass
```
Comment (dougbu) [inline src/SignalR/clients/java/signalr/core/build.gradle:81]: Didn't think I set VS Code to add trailing newlines _everywhere_. Will this cause problems @BrennanConroy❔

**Review by wtgodbe — APPROVED** [review]

**Comment by dougbu** [comment]
Note: This will go in **after** I rename the branch. In a sense, it'll test out the auto-magic PR handling😁

**Comment by pranavkm** [comment]
FYI @sebastienros

File: src/SignalR/clients/java/signalr/core/build.gradle (hunk 1 above)
Comment (BrennanConroy) [reply src/SignalR/clients/java/signalr/core/build.gradle:81]: Hope not, we'll see if the CI fails :D

**Review by pranavkm — APPROVED** [review]

File: src/Shared/runtime/ReadMe.SharedCode.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ dotnet/AspNetCore code paths:
 - `PS D:\github\runtime\src\libraries\System.Net.Http\tests\UnitTests> dotnet build /t:test`
 
 ## Building dotnet/AspNetCore code:
-- https://github.com/dotnet/AspNetCore/blob/master/docs/BuildFromSource.md
+- https://github.com/dotnet/aspnetcore/blob/main/docs/BuildFromSource.md
```
Comment (Tratcher) [inline src/Shared/runtime/ReadMe.SharedCode.md:27]: This file must be updated in runtime as well or the bot will overwrite this change.

**Review by Tratcher — APPROVED** [review]

File: src/Shared/runtime/ReadMe.SharedCode.md (hunk 2 above)
Comment (dougbu) [reply src/Shared/runtime/ReadMe.SharedCode.md:27]: Thanks @Tratcher❕ See dotnet/runtime#47306

**Pull request merged**